NitroxServer/Serialization/BatchCellsParser.cs

```diff
@@ -80,8 +80,7 @@ public List<EntitySpawnPoint> ParseBatchData(Int3 batchId)
 
         public void ParseFile(Int3 batchId, string pathPrefix, string suffix, List<EntitySpawnPoint> spawnPoints)
         {
-            // This isn't always gonna work.
-            string path = @"C:\Program Files (x86)\Steam\steamapps\common\Subnautica\SNUnmanagedData\Build18\";
+            string path = Path.Combine(SteamFinder.FindSteamGamePath(264710, "Subnautica"), "SNUnmanagedData/Build18") + "/";
```

Comment:
We should detect the path only once in the code, allowing us to overwrite it later based on a config file, if desired.

Reply:
Sure, I just don't know where to put it where both classes to reference it. Static string seems to hacky, however it is true the path will never change.

Reply:
That's solved once we have an IoC container as mentioned by @MadMaxOfYore in #101. This allows us to create a singleton class responsible for providing the settings from a configuration file (and sensible default values), which is then accessible through the container from everywhere else in the program without explicitly passing it around.

Reply:
Yeah - something like this would be a great candidate to roll up within a ConfigurationProvider. However; given the game world that we live in, any sort of centralized ConfigurationProvider may need to take too many references and could end up being a pretty nasty God class. 

In this minimalist environment, it may be prudent to simply roll an ISteamPathProvider that carries the responsibility of providing this information on demand. The ISteamPathProvider abstraction need not be implemented in isolation. Rather, we could write a facade called SteamProvider that implements multiple interfaces. Everyone would reference ISteamPathProvider but would be provided with an instance of SteamProvider after resolving the dependency from an IoC container.

This isn't something that I'd expect nes to address right now though. I'd say let it roll as-is until we can get an IoC implementation prioritized. For now - I think commenting a TODO in the code that addresses the tech-debt and intended solution would be "good enough" for this PR.